test/test_torch.py

```diff
@@ -12520,6 +12520,14 @@ def test_memory_format(self):
         self.assertTrue(nhwc.is_contiguous(memory_format=torch.channels_last))
         self.assertEqual(nhwc, x)
 
+    def test_memory_format_consistency(self):
```

Comment:
Wait, does this test fails if we remove all `TensorImpl.h` changes

Reply:
Yeah, that's the issue I was complaining (along with the ambiguity things) earlier last week.

I guess I didn't make it clear enough back then. :)